xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1223,13 +1223,15 @@ void CVideoPlayer::Prepare()
   if (std::shared_ptr<CDVDInputStream::IMenus> ptr = std::dynamic_pointer_cast<CDVDInputStream::IMenus>(m_pInputStream))
   {
     CLog::Log(LOGINFO, "VideoPlayer: playing a file with menu's");
-    if(std::dynamic_pointer_cast<CDVDInputStreamNavigator>(m_pInputStream))
```

Comment:
Just a note on this one. We do not save the player state anymore if we are not in a vts domain (playing a dvd title), so this was forcing the resumepoint to be ignored. Tracking and relying on libdvdnav time is not reliable.